broker/broker_utils/setup.py

```diff
@@ -30,8 +30,8 @@
     requirements = f.read().splitlines()
 
 setup(
-    name='pgb_broker_utils',  # Required
-    version='0.2.10',  # Required
+    name='pgb_broker_utils',
+    version='0.2.20',
```

Comment:
Is this key line where the version of the broker is tracked?  Consider doing some reading into how people handle this in general.  It seems like having some more visible file that defines version and then is imported into things that need the version might be a way to go here.

Reply:
But also, I'm starting to wonder, if this is specifically the `pgb_broker_utils` version, that implies that we are tracking versions separately.  If we're going to have separate versions, should these be in separate repos?

Reply:
> It seems like having some more visible file that defines version and then is imported into things that need the version might be a way to go here.

Makes sense, and I have seen others do something similar. I'll look at other examples more closely.

> But also, I'm starting to wonder, if this is specifically the `pgb_broker_utils` version, that implies that we are tracking versions separately. 

Correct, and see my (long) comment below for more discussion.

> If we're going to have separate versions, should these be in separate repos?

Addressed below.